lib/prawn/svg/elements/gradient.rb

```diff
@@ -92,41 +92,43 @@ def load_gradient_configuration
     @units = derive_attribute('gradientUnits') == 'userSpaceOnUse' ? :user_space : :bounding_box
 
     if (transform = derive_attribute('gradientTransform'))
-      @transform_matrix = parse_transform_attribute(transform)
+      @transform_matrix = parse_transform_attribute(transform, space: :svg)
     end
 
-    if (spread_method = derive_attribute('spreadMethod')) && spread_method != 'pad'
-      warnings << "prawn-svg only currently supports the 'pad' spreadMethod attribute value"
+    if (spread_method = derive_attribute('spreadMethod'))
+      @wrap = spread_method.to_sym
```

Comment:
Might be good here to check whether it's a valid value (`pad`, `reflect`, `repeat`) and only set `@wrap` if it's one of these?

Reply:
Yes, good idea. I have pushed a change to do that.